transformations/transformations/single_column_coalesced.py

```diff
@@ -116,8 +116,8 @@ def get_integer_variable(cls, routine, name):
         name : string
             Name of the variable to find the in the routine.
         """
-        if name in routine.variable_map:
-            v_index = routine.variable_map[name]
+        if name in routine.symbol_map:
+            v_index = routine.symbol_map[name]
         else:
```

Comment:
Since constructing these maps is expensive, could we maybe do it like this?
```suggestion
        v_index = routine.symbol_map.get(name)
        if v_index is None:
```